nano/templates/food_replicator.tmpl

```diff
@@ -0,0 +1,105 @@
+<div class='item'>
+	{{:helper.link('Close', 'gear', {'close' : 1}, null, 'fixedLeft')}}
+</div>
+
+<div class="item">
+	<div class="itemLabel">
+		Food Paste Stored:
+	</div>
+	<div class="itemContent">
+		{{:data.points}} Units
+	</div>
+</div>
+<div class="item">
+	{{if data.loaded}}
+	<div class="itemLabel" style="width: 70%">
+		{{:data.loaded.name}}:
+	</div>
+	<div class="item">
+		{{if data.loaded.recycle_value >= 0}}
+			Recycle Value: <span class='average'>{{:data.loaded.recycle_value}}</span>
+			{{:helper.link('Recycle', 'recycle', { "recycle" : 1 })}}
+			<br>
+		{{/if}}
+		{{if data.loaded.duped}}
+			<span class='bad'>Replication Degradation Detected.</span>
+		{{else}}
+			Replication Cost: <span class='average'>{{:data.loaded.rep_cost}} </span>
+			{{if data.points >= data.loaded.rep_cost}}
+				{{:helper.link('Replicate', 'clone', { "replicate" : 5 })}}
+			{{else}}
+				<span class='bad'>Insufficient Food Paste.</span>
+			{{/if}}
+		{{/if}}
+		<br>
+		{{:helper.link('Eject', 'eject', { "eject" : 1 })}}
+	</div>
+	{{else}}
+		<span class='average'>No dish loaded.</span>
+	{{/if}}
+</div>
+<div class='item'>
+	{{if data.memory}}
+		{{for data.memory}}
```

Comment:
I could make it not show locked memory slots, I just thought it would be better to communicate to the players that they can unlock more slots than make them magically appear.